plan/typeinfer_test.go

```diff
@@ -1666,3 +1667,18 @@ func (s *testPlanSuite) createTestCase4LikeFuncs() []typeInferTestCase {
 		{"c_enum        regexp c_text_d", mysql.TypeLonglong, charset.CharsetBin, mysql.BinaryFlag, 1, 0},
 	}
 }
+
+func (s *testPlanSuite) createTestCase4JSONFuncs() []typeInferTestCase {
+	return []typeInferTestCase{
+		{"json_type(c_json)", mysql.TypeVarString, charset.CharsetUTF8, 0, 0, types.UnspecifiedLength},
+		{"json_unquote(c_json)", mysql.TypeVarString, charset.CharsetUTF8, 0, 0, types.UnspecifiedLength},
```

Comment:
seems like `Flen` in return type of function `json_type` and `json_unquote` is not correct, should not be 0.